docs/governance/overview.md

```diff
@@ -19,54 +19,94 @@ This page provides an overview of the On-chain Governance module that will be av
 
 ## Overview
 
-The MultiversX network is able to handle on-chain governance proposes by issuing special types of transactions. This was implemented as a mean for further increasing the decentralization of the decision-making process.
+The MultiversX network enables on-chain governance by issuing special types of transactions. This mechanism increases decentralization by allowing community members to directly propose and vote on changes, such as protocol upgrades or configuration adjustments.
 
-Anyone can create a proposal using their wallet and issuing a special type of transaction specifying an identifier, a starting epoch and an end epoch.
-
-Users that staked EGLD will be able to cast votes upon opened proposals. The voting power is proportional with the staked value and is computed as `voting_power = staked_value` (linear voting).
+- **Anyone** can create a proposal by paying the proposal fee and specifying:
+  - a **commit hash** (unique identifier, usually a GitHub commit or spec reference)
+  - a **start epoch** and an **end epoch** (the voting window).
+- **Any staked user** (direct or delegated) can vote during the active period.
+- Voting power is proportional to stake:  
+  `voting_power = staked_value` (linear voting).
 
 [comment]: # (mx-context-auto)
 
 ## Implementation details
 
-Each proposal costs 1000 EGLD that will be locked during the voting of the proposal. If the proposal is either rejected or accepted, the entire locked sum can be unlocked & withdrawn. If the proposal do not pass, 10 EGLD will remain locked in the governance smart contract as a penalty fee. The proposal costs around 51 million of gas units.
+### Proposals
 
-There are 4 types of votes: **Yes**, **No**, **Abstain** and **Veto**. Each of the vote costs around 6 million of gas units.
+In order for a proposal to be submitted, the following requirements need to be met:
+- For a period of at least 15 days, the proposal needs to be published on Agora for public debate and analysis;
+- Each proposal requires paying a `ProposalFee` (currently **500 EGLD**);
+- Each proposal costs around 51 million of gas units to be submitted.
+- The starting epoch of the proposal's voting period needs to be set inside an interval of 30 epochs from the epoch in which the proposal was submitted;
 
-A user can create any number of proposals as long as it pays the locking fee & the gas used for the transaction. The proposal can be closed in any epoch following the provided end epoch.
+After a proposal is created, the following rules will apply: 
+- If the proposal passes or fails normally, the fee is refunded to the issuer.  
```

Comment:
if the proposal fails, we currently refund the 500 from the proposal fee without the `LostProposalFee` so only if the proposal passes, the entire fee is refunded.